src/alliteratedActors.js

```diff
@@ -22,7 +22,14 @@
  * @return {string}
  */
 function alliteratedActors(names) {
-  // write code here
+  const arrayOfNames = names.map(name => name.split(' '));
+  const result = arrayOfNames.filter(fullName =>
+    fullName.every(([firstLetter]) => firstLetter === fullName[0][0])
```

Comment:
wrap this in brackets and leave opening bracket on the prev line and closing one on the next line